Documentation/dev/deploy-metering.md

```diff
@@ -0,0 +1,165 @@
+# Overview
+
+The `deploy-metering` utility aims to emulate the install/uninstall logic that's currently done in the `hack/install.sh` and `hack/uninstall.sh` shell scripts.
+
+When interacting with the `deploy-metering` binary, you have the choice of using the command flags available, or their respective environment variable.
+
+In the case where both the flag, and the environment variable are specified, the [flag value takes precedence](#example-of-precedence) over the value stored in the environment variable.
+
+## Building the binary
+
+```bash
+make bin/deploy-metering
+```
+
+### Prerequisites
+
+With the addition of the deploy-metering CLI, we can assign a default (typically an empty value) value for most of the configuration options.
+
+This means the only required flags/environment variables that needs to be set, are the namespace in which the Metering stack will be managed and the path to the `MeteringConfig` custom resource file.
+
+Before deploying metering, ensure that either of the following options are set:
+
+```bash
+export METERING_NAMESPACE="metering"
```

Comment:
@timflannagan1 OLM install via OperatorHub defaults to `openshift-metering`.  Perhaps change the value to match

Reply:
My intent was to make it platform agnostic - once we start adding a better OLM installation path to this utility (right now it only installs via the redhat-operators catalogsource), we may want to start documenting a more restrictive namespace recommendation, at least for Openshift.